lib/quby/questionnaires/entities/fields.rb

```diff
@@ -54,6 +46,16 @@ def add(question)
           end
         end
 
+        def check_key_clashes(new_answer_keys, new_input_keys)
+          if @answer_keys.intersect?(new_answer_keys)
+            fail "Duplicate answer keys: #{@answer_keys.intersection(new_answer_keys).inspect}"
+          end
+
+          if @input_keys.intersect?(new_input_keys)
+            fail "Duplicate input keys: #{@input_keys.intersection(new_input_keys).inspect}"
+          end
+        end
```

Comment:
Als je het doet voor answer- en inputkeys, is het dan niet logisch om ook te doen voor scorekeys?